generators/cpp_client.py

```diff
@@ -2756,7 +2756,7 @@ def _c_accessors_list(self, field):
                 (cookie_type, self.c_reply_name, base_func_name))
     f.write('.SH ERRORS\n')
     if hasattr(self, "doc") and self.doc:
-        for errtype, errtext in self.doc.errors.items():
+        for errtype, errtext in list(self.doc.errors.items()):
```

Comment:
If I'm not wrong this is wrong no?
This transform a dict view into a list.
```suggestion
        for errtype, errtext in self.doc.errors.items():
```